node-runners/start/README.md

```diff
@@ -262,6 +264,14 @@ Number of shards that the target network is partition in. For the Alpha network
 
 Universe identity and properties \(such as gossip port\). This string separates two Radix DLT networks from each other.
 
+### CORE\_SECURE\_RANDOM\_SOURCE <a id="core_secure_random_source"></a>
+
+Secure random number device used by the JVM. The default device is [/dev/urandom](https://linux.die.net/man/4/urandom).
+However, power users might want to switch to `/dev/random`, in which case a decent a entropy generator side-car container
```

Comment:
remove an 'a', in this line; changed to "... in which case a decent entropy generator ..."